README.md

```diff
@@ -1,107 +1,196 @@
-
-  app.get('/streets/:streetId/buildings/:id', (req, res) => {
-    const buildingsOnStreet = buildings.filter(({ streetId }) => 
-      streetId === req.params.streetId);
-    res.json(buildingsOnStreet.find(({ id }) => id === req.params.id));
-  });
-};
+// app.js
+var express = require('express');
+var app = express();
+var pingPongRouter = require('./ping-pong');
+
+app.get('/', function(req, res) {
+  res.send('Hello World!');
+});
+
+app.get('/hello', function(req, res) {
+  res.send('Hello hello!');
+});
+
+app.get('/hello/:name', function(req, res) {
+  res.send('Hello ' + req.params.name);
+});
+
+app.use('/ping-pong', pingPongRouter);
+
+app.listen(3000, function() {
+  console.log('Express app listening on port 3000!');
+});
 ```
 
+Now try running `curl localhost:3000/ping-pong/ping` and `curl localhost:3000/ping-pong/pong`.
+
+Why should we get so excited about moving some routes into a separate router? For one it helps keep our files smaller and more contained. Rather than having all of our routes in one file, we can have small files with a a group of routes.
+
+Another reason is now we can easily _mount_ our router on any prefix. Let's say instead of wanting our paths to be `/ping-pong/ping` and `/ping-pong/pong`, we wanted them to be `/superfungame/ping` and `/superfungame/pong`. Great news, we only have to change one small part of our code! See `app.js` below to see the change.
+
 ```js
 // app.js
+var express = require('express');
+var app = express();
+var pingPongRouter = require('./ping-pong');
+
+app.get('/', function(req, res) {
+  res.send('Hello World!');
+});
+
+app.get('/hello', function(req, res) {
+  res.send('Hello hello!');
+});
 
-const express = require('express');
-const streetRoutes = require('./routes/streets').router;
-const buildingRoutes = require('./routes/buildings').router;
+app.get('/hello/:name', function(req, res) {
+  res.send('Hello ' + req.params.name);
+});
 
-// boilerplate ...
+// All we had to do was change ping-pong to superfungame!
+app.use('/superfungame', pingPongRouter);
 
-streetRoutes(app);
-buildingRoutes(app);
+app.listen(3000, function() {
+  console.log('Express app listening on port 3000!');
+});
 ```
 
-(This is a janky, untested example — there's a better way to organize this stuff.)
+If we structure our routes with routers, it will be very easy for us change what paths cause our routing functions to be called.
+
+## Conclusion
+
+In this lesson we learned how to start building an application with many routes. We say how we can capture parts of the path; the same technique that Facebook uses to create a separate url for each of their users. We also saw that we can create a separate router using `express.Router` which we can then mount in into our Express app.
 
-But the point is we should walk students through this sort of thing.
+With this foundation we are now ready to start manipulating data in the [next lesson]( https://github.com/learn-co-curriculum/express-routing-crud
```

Comment:
These lessons might not always be in this order -- in fact, we often end up reusing certain lessons (and not complementary ones) in other curriculum -- so we might want to make this ending a bit more generic.